src/EnergyPlus/UnitarySystem.cc

```diff
@@ -11044,10 +11047,8 @@ namespace UnitarySystems {
         if (this->m_EMSOverrideCoilSpeedNumOn) {
             // pass
         } else {
-            auto const SELECT_CASE_var(this->m_CoolingCoilType_Num);
-
-            if (SELECT_CASE_var == DataHVACGlobals::CoilDX_CoolingSingleSpeed) { // Coil:Cooling:DX:SingleSpeed
-
+            switch (this->m_CoolingCoilType_Num) {
```

Comment:
I am just going to throw this out there. I am fine with using `m_` to refer to member variables (it's a scope thing, not a type thing so technically not "Hungarian"). I am somewhat fine with using `this->` to indicate reference to member variables. I am not sure what using `this->m_` accomplishes. This is a member variable. Also, this is a member variable in case you didn't hear me the first time.